db/migrations/postgres/000063_create_identities_table.up.sql

```diff
@@ -0,0 +1,137 @@
+BEGIN;
+
+CREATE TABLE identities (
+  seq                   SERIAL          PRIMARY KEY,
+  id                    UUID            NOT NULL,
+  did                   VARCHAR(256)    NOT NULL,
+  parent                UUID,
+  messages_claim        UUID            NOT NULL,
+  messages_verification UUID,
+  messages_update       UUID,
+  itype                 VARCHAR(64)     NOT NULL,
+  namespace             VARCHAR(64)     NOT NULL,
+  name                  VARCHAR(64)     NOT NULL,
+  description           VARCHAR(4096)   NOT NULL,
+  profile               TEXT,
+  created               BIGINT          NOT NULL,
+  updated               BIGINT          NOT NULL
+);
+
+CREATE UNIQUE INDEX identities_id ON identities(id);
+CREATE UNIQUE INDEX identities_did ON identities(did);
+CREATE UNIQUE INDEX identities_name ON identities(itype, namespace, name);
+
+CREATE TABLE verifiers (
+  seq            SERIAL          PRIMARY KEY,
+  id             UUID            NOT NULL,
+  identity       UUID            NOT NULL,
+  vtype          VARCHAR(256)    NOT NULL,
+  namespace      VARCHAR(64)     NOT NULL,
+  value          TEXT            NOT NULL,
+  created        BIGINT          NOT NULL
+);
+
+CREATE UNIQUE INDEX verifiers_id ON verifiers(id);
+CREATE UNIQUE INDEX verifiers_value ON verifiers(vtype, namespace, value);
+CREATE UNIQUE INDEX verifiers_identity ON verifiers(identity);
+
+INSERT INTO identities (
```

Comment:
This is obviously more complex than all our previous migrations, as we merge two tables into one.

I have managed to make it so we delete the original tables, and the down migration re-creates them - but only with data that exists prior to the up-migration.